dockerfiles/che/entrypoint.sh

```diff
@@ -262,8 +262,6 @@ init() {
   sed -i "/che.workspace.storage=/c\che.workspace.storage=/data/workspaces" $CHE_LOCAL_CONF_DIR/che.properties
   sed -i "/che.database=/c\che.database=/data/storage" $CHE_LOCAL_CONF_DIR/che.properties
   sed -i "/che.template.storage=/c\che.template.storage=/data/templates" $CHE_LOCAL_CONF_DIR/che.properties
-  sed -i "/che.stacks.storage=/c\che.stacks.storage=/data/stacks/stacks.json" $CHE_LOCAL_CONF_DIR/che.properties
-  sed -i "/che.stacks.images=/c\che.stacks.images=/data/stacks/images" $CHE_LOCAL_CONF_DIR/che.properties
```

Comment:
@riuvshin yes, we moved all stacks images [into](https://github.com/eclipse/che/tree/bed4280261d773887f8101e000ca103d983a5525/ide/che-core-ide-stacks/src/main/resources/stacks-images)